bower.json

```diff
@@ -52,7 +52,8 @@
     "promise-polyfill": "^5.2.1",
     "font-awesome": "fontawesome#^4.7.0",
     "fetch": "^2.0.2",
-    "file-saver": "^1.3.3"
+    "file-saver": "^1.3.3",
+    "vimeo-player-js": "^2.1.0"
```

Comment:
Adding to vendor would also just put it in the giant vendor file. You can use the ModuleLoader to load this dynamically by adding it [here](https://github.com/codecombat/codecombat/blob/paypal-subs/config.coffee#L124-124) to not include it in `vendor.js` and [here](https://github.com/codecombat/codecombat/blob/paypal-subs/config.coffee#L155) to copy the file to the public folder. To load it in a given view, require it like [this](https://github.com/codecombat/codecombat/blob/paypal-subs/app/views/editor/level/LevelEditView.coffee#L42), and the ModuleLoader will take care of the rest with magic.